tools/DataGen.py

```diff
@@ -0,0 +1,38 @@
+"""
```

Comment:
If this file is used for test, please place it under the `tests` folder, not in `tools`